frontend/src/components/UserInfoCard.tsx

```diff
@@ -86,27 +86,27 @@ export const UserInfoCard = ({ userInfo }: { userInfo: UserInfo }) => {
   const emailOnOk = async (newValue: string) => {
     dispatch(
       modalsActions.showModal({
-        modalType: ModalTypes.CONFIRM_PASSWORD_MODAL,
+        modalType: ModalTypes.CHANGE_PASSWORD_MODAL,
         modalProps: {
           actionData: { id: userInfo.id, email: newValue },
         },
       }),
     );
   };
-  const passwordOnOk = async (newValue: string) => {
+  const passwordOnOk = async () => {
```

Comment:
The name `passwordOnOk` doesn't really reflect the current function
maybe `changePassword`, `openChangePasswordModal`, or something like that

Also the `async` is no longer necessary, it was just required by the `EditableTextWithButtons` component
```suggestion
  const changePassword = () => {
```

Reply:
You are right, name changed.